static/new_home/style.css

```diff
@@ -84,6 +84,7 @@ nav
   margin: 10px;
   color: #fff;
   background: #eb5d1e;
+  cursor: pointer;
```

Comment:
It was causing problem because the button was wrapped being wrapped by a `p` tag instead of an `a` tag. I have fixed this in the latest commit, so you can remove this from here.